### DISCUSSION TITLE
Blank page and MIME type error when running SampleApp with ReactPy on Python 3.10.5 and Windows 10

### DISCUSSION THREAD

**Discussion by sn0vv** [idom-team/idom#1074] [Problem]
Hello,

I install reactpy on Python 3.10.5 and Windows 10.

> pip install "reactpy[starlette]"

I run the sample application.
> python -c "import reactpy; reactpy.run(reactpy.sample.SampleApp)"

But Chrome shows a blank page and DevTools Console outputs the following:

> index.a1f87a11.js:1 Failed to load module script: Expected a JavaScript module script but the server responded with a MIME type of "text/plain". Strict MIME type checking is enforced for module scripts per HTML spec.

Visual Studio Code TERMINAL outputs the following:

> (venv_react) PS D:\dev\python\reactpy\venv_react> python -c "import reactpy; reactpy.run(reactpy.sample.SampleApp)"
> 2023-06-19T22:23:33+0900 | WARNING | The `run()` function is only intended for testing during development! To run in production, consider selecting a supported backend and importing its associated `configure()` function from `reactpy.backend.<package>` where `<package>` is one of ['starlette', 'fastapi', 'sanic', 'tornado', 'flask']. For details refer to the docs on how to run each package.
> 2023-06-19T22:23:33+0900 | INFO | Running with starlette.applications.Starlette at http://127.0.0.1:8000
> INFO:     Started server process [31804]
> INFO:     Waiting for application startup.
> INFO:     Application startup complete.
> INFO:     Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)
> INFO:     127.0.0.1:63690 - "GET / HTTP/1.1" 200 OK
> INFO:     127.0.0.1:63690 - "GET /index.a1f87a11.js HTTP/1.1" 304 Not Modified

Please let me know what is wrong.
Thanks in advance.

**Comment by rmorshea** [comment]
Duplicate of https://github.com/reactive-python/reactpy/discussions/1073
